### ISSUE TITLE
Do we want factory default settings for NVDA?

### ISSUE THREAD

**Issue by robfentress** [w3c/aria-at#391]
On the [Configuring Screen Readers for Testing](https://github.com/w3c/aria-at/wiki/Configuring-Screen-Readers-for-Testing), testers are instructed to revert to default settings before running a test.  I am concerned that for NVDA this may lead to inconsistent results, especially for those performing these tests who may be sighted, since "Enable mouse tracking" is a part of the default configuration for NVDA.  If the user inadvertently moves the mouse over another element while performing a test, the screen reader will say what is under the cursor, which may lead to confusion for those not used to this behavior.

**Comment by jscholes** [comment]
It was determined during the May 27th community group meeting that training for testers will prepare for this; testers should be sufficiently aware of screen reader behaviour to avoid filing misleading results e.g. caused by accidental mouse movements.
